weblate/utils/tests/test_render.py

```diff
@@ -36,3 +36,8 @@ def test_float(self):
     def test_float_cs(self):
         with override('cs'):
             self.test_float()
+
+    def test_replace(self):
+        result = replace('a-string-with-dashes', '-', ' ')
+        self.assertEqual(result, 'a string with dashes')
+
```

Comment:
Can you please test this through render_template, so that we know that the tag is registered there?